### PR TITLE
Add Verse (Bitcoin.com) to Projects

### PR DESCRIPTION
**Pull request by lizzydude90**

### CODE REVIEW

**Comment by 0xBoxer** [comment]
@philippWassibauer would appreciate a review

File: pages/verse/verse.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  - "Henrystats/verse-farmers-analytics"
+
+# List of relevant query-ids.
+queries:
+  - 1942119
+
+# List of visualizations to embed
+embeds:
+  - name: “Verse Holders over time”
+    link: "https://dune.com/queries/1942119/3204019"
```
Comment (philippWassibauer) [inline pages/verse/verse.md:54]: this needs to be an embed, eg: 
`https://dune.com/embeds/31193/62877/a7284a40-96e1-484b-a99b-cc00c25f16b5`

**Review by philippWassibauer — CHANGES_REQUESTED** [review]
if the embed is fixed this is good to go.

File: pages/verse/verse.md (hunk 1 above)
Comment (philippWassibauer) [inline pages/verse/verse.md:54]:
```suggestion
    link: "https://dune.com/embeds/1942119/3204019" 
```

**Review by philippWassibauer — APPROVED** [review]

**Comment by philippWassibauer** [comment]
@0xBoxer @lizzydude90 I had to remove the embed, as it broke the page. You need to add the secret to the embed, like so: 

https://dune.com/embeds/2452344/4031032/4b581815-f7ec-485f-8c4e-9556c5a05eb2

Easiest done by creating a private query and then clicking embed. Please submit another PR to fix this.

**Comment by lizzydude90** [comment]
Thanks Phillip,

We're going to submit another update next week to include the latest
suggestions, will include your suggestions. Much appreciated.

Have a good weekend,

**Pull request merged**